src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -128,16 +137,13 @@ const NavHeader = (props) => {
               </View>
             </>
           )}
-          <View
-            style={{
-              flexDirection: 'row',
-              alignItems: 'flex-start',
-              paddingTop: insets.top,
-              zIndex: 2,
-              position: 'absolute'
-            }}>
-            <TouchableOpacity onPress={() => backScreen()} style={styles.backbutton}>
-              <MemoIcArrowBackCircle width={normalize(32)} height={normalize(32)} />
+        </TouchableOpacity>
+        <View style={styles.containerAction}>
+          {!isFollow && isHeaderHide ? (
+            <ButtonFollow handleSetFollow={handleFollowTopic} />
+          ) : (
+            <TouchableOpacity onPress={onShareCommunity} style={styles.shareIconStyle}>
+              <ShareIconCircle color="black" width={32} height={32} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering logic has been changed here. Make sure that the new conditions correctly represent the intended behavior for when to show the follow button and when to show the share icon.



<!-- This is an auto-generated comment by OSS CodeRabbit -->